test/treegen_test.lua

```diff
@@ -0,0 +1,86 @@
+local t = require('luatest')
+local fio = require('fio')
+
+local treegen = require('luatest.treegen')
+
+local g = t.group()
+
+
+g.test_init = function()
+    treegen.init(g)
+
+    t.assert_equals(g.tempdirs, {})
+    t.assert_equals(g.templates, {})
+end
+
+g.after_test('test_init', function()
+    treegen.clean(g)
+end)
```

Comment:
I think that init/clean are private functions: they shouldn't be exported from the module and they shouldn't be tested.